### PR TITLE
GTWint bug and variable names fixed

### PR DESCRIPTION
**Pull request by mbogor**
#### What's this PR do?
fixes bug and naming conventions in the original GTW integration feature
#### Where should the reviewer start?
jobs/scheduling/AttendanceSync.js
#### How should this be manually tested?
the usual, don't forget to comment out the mandrill call so that heather doesn't get overloaded w duplicate emails
#### What are the relevant tickets?
INT-192

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -60,8 +60,10 @@ CONFIG.wiwAccountID = {
   supervisors: 622060,
 };
 
-//in seconds
+//used in AttendanceSync.js to check whether trainees attended a GTW webinar for a minimum of 90 minutes = 5400 seconds
 CONFIG.GTW_attendance_minimum = 5400;
+//used in AttendanceSync.js to set the time range over which we query GTW sessions
+CONFIG.GTW_time_range_query = 72;
```
Comment (jrauschenberg) [inline config.js]: Time range in what? Hours?

File: jobs/scheduling/AttendanceSync.js [hunk 2]
```diff
@@ -6,13 +6,13 @@ var mandrill = require('mandrill-api/mandrill');
 var mandrill_client = new mandrill.Mandrill(KEYS.mandrill.api_key);
 var fs = require('fs');
 
-//runs crone job every day at 5am
+// //runs crone job every day at 5am
 new CronJob(CONFIG.time_interval.gtw_attendance_sync_with_canvas, function () {
```
Comment (jrauschenberg) [inline jobs/scheduling/AttendanceSync.js]: Might want to change the double "//" here, and "crone" to "cron."

**Comment by jrauschenberg** [comment]
A couple minor comments, otherwise looks good @mbogor!

**Comment by tongxiang** [comment]
👍

**Pull request merged**